boards/riscv/qemu_riscv64/qemu_riscv64_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_RISCV=y
+CONFIG_SOC_SERIES_RISCV_SIFIVE_FREEDOM=y
+CONFIG_SOC_RISCV_SIFIVE_FREEDOM=y
+CONFIG_BOARD_QEMU_RISCV64=y
+CONFIG_64BIT=y
+CONFIG_CONSOLE=y
+CONFIG_PRINTK=y
+CONFIG_SERIAL=y
+CONFIG_UART_SIFIVE=y
+CONFIG_UART_SIFIVE_PORT_0=y
+CONFIG_UART_CONSOLE=y
+CONFIG_PLIC=y
+CONFIG_PINMUX=y
+CONFIG_PINMUX_SIFIVE=y
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_GPIO=y
+CONFIG_GPIO_SIFIVE=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=10000000
+CONFIG_STACK_SENTINEL=y
+CONFIG_TEST_EXTRA_STACKSIZE=1024
```

Comment:
Any reason for `CONFIG_STACK_SENTINEL` & `CONFIG_TEST_EXTRA_STACKSIZE`?  If I remove these all the tests you flag as `platform_exclude: qemu_riscv64` build.